freeipa/src/main/java/com/sequenceiq/freeipa/controller/UserV1Controller.java

```diff
@@ -22,6 +13,16 @@
 import com.sequenceiq.freeipa.service.freeipa.user.PasswordService;
 import com.sequenceiq.freeipa.service.freeipa.user.SyncOperationStatusService;
 import com.sequenceiq.freeipa.service.freeipa.user.UserService;
+import com.sequenceiq.freeipa.util.CrnService;
```

Comment:
unnecessary re-ordering of imports.

Reply:
why un-necessary? And moreover its due to settings from CDP and CB. I cant do much there.